plugins/html-plugin/src/main/java/ru/sbtqa/tag/pagefactory/find/HtmlFindUtils.java

```diff
@@ -0,0 +1,382 @@
+package ru.sbtqa.tag.pagefactory.find;
+
+import static java.lang.String.format;
+import static ru.sbtqa.tag.pagefactory.find.ComplexElement.ELEMENT_SEPARATOR;
+import java.lang.reflect.Field;
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import ru.sbtqa.tag.pagefactory.Page;
+import ru.sbtqa.tag.pagefactory.annotations.ElementTitle;
+import ru.sbtqa.tag.pagefactory.context.PageContext;
+import ru.sbtqa.tag.pagefactory.environment.Environment;
+import ru.sbtqa.tag.pagefactory.exceptions.ElementDescriptionException;
+import ru.sbtqa.tag.pagefactory.exception.ElementSearchException;
+import ru.sbtqa.tag.pagefactory.exception.IncorrectElementTypeException;
+import ru.sbtqa.tag.pagefactory.utils.HtmlElementUtils;
+import static ru.sbtqa.tag.pagefactory.utils.HtmlElementUtils.createElementWithCustomType;
+import ru.sbtqa.tag.pagefactory.utils.Wait;
+import ru.sbtqa.tag.qautils.errors.AutotestError;
+import ru.sbtqa.tag.qautils.reflect.FieldUtilsExt;
+import static ru.yandex.qatools.htmlelements.utils.HtmlElementUtils.isHtmlElement;
+import static ru.yandex.qatools.htmlelements.utils.HtmlElementUtils.isTypifiedElement;
+
+public class HtmlFindUtils extends FindUtils {
+
+    @Override
+    @Deprecated
+    public <T> T getElementByTitle(Page page, String title) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1133))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=133&issues=AWfPt2o64tUZZzFFj6HY&open=AWfPt2o64tUZZzFFj6HY)

[ISSUE_KEY]: AWfPt2o64tUZZzFFj6HY